packages/block-editor/src/components/copy-handler/README.md

```diff
@@ -0,0 +1,42 @@
+# Copy Handler
+
+The `CopyHandler` component handles the copy, cut and paste events of its children blocks. It handles multiple selection of blocks as well.
+
+Concretely, it handles the display of success messages and takes care of copying the block to the clipboard using the [raw handling API](https://github.com/WordPress/gutenberg/tree/master/packages/blocks/src/api/raw-handling).
+
+![Copy cut behaviours](https://make.wordpress.org/core/files/2020/09/copy-cut-behaviour.gif)
+
+## Table of contents
+
+1. [Development guidelines](#development-guidelines)
+2. [Related components](#related-components)
+
+## Development guidelines
+
+### Usage
+
+Adds the copy handler functionnalities to the editor.
```

Comment:
```suggestion
Adds block-level copy-and-paste support to the editor.
```

Reply:
It's important that this distinction be clear, as `CopyHandler` doesn't handle (and isn't required for) copying and pasting of fragments of content within a block. Those will be natively handled (e.g. selecting a piece of the block UI and copying, or selecting a fragment of RichText and copying).